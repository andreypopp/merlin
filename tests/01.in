["find", "use", "sexplib"]
["tell", "struct", "module Test_sexp = struct "]
["tell", "struct", "type t = int with sexp "]
["tell", "struct", "end"]
["tell", "struct", null]
["errors"]
["dump", "env"]
