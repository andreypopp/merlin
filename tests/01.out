["return",true]
["return",false]
["return",false]
["return",false]
["return",true]
["return",[]]
["return",[["File \"\", line 1, characters 31-39","module Test_sexp :\n  sig\n    type t = int\n    val sexp_of_t : t -> Sexplib.Sexp.t\n    val t_of_sexp : Sexplib.Sexp.t -> t\n  end"]]]
